site/content/en/references/kustomize/cmd/localize/_index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "kustomize localize"
+linkTitle: "localize"
+type: docs
+weight: 9
+description: >
+    Replace urls with local paths
+---
+
+Disclaimer: This is an alpha command. Please see the [command proposal](https://github.com/kubernetes-sigs/kustomize/blob/master/proposals/22-04-localize-command.md)
+for full capabilities.
+
+### Feedback
+
+Please leave your feedback for this command under [the following issue](https://github.com/kubernetes-sigs/kustomize/issues/4996).
+
+### Man
+
+The `kustomize localize` command makes a recursive copy of a kustomization
+in which referenced urls are replaced by local paths to their downloaded content.
+
+The purpose of this command is to create a copy on which
+`kustomize build`<sup>[[build]](#notes)</sup> produces the same output
+without a network connection.
+
+The command takes the following form:
+
+<pre>
+<b>kustomize localize</b> [<ins>target</ins> [<ins>newDir</ins>]] [--scope <ins>scope</ins>]
+</pre>
+
+where
+
+* `target` is the [kustomization directory](https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#kustomization-root) 
+to localize. This value can be a local path or a [remote root](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md). 
+The default value is the current working directory.
+* `newDir` is the destination of the "localized" copy that the command
+will create. The destination must reside in an existing directory.
+The default destination is a directory in the current working directory named:
+  * `localized-{target}` for local `target`
+  * `localized-{target}-{ref}`<sup>[[ref]](#notes)</sup> for remote `target`
+* `scope` is the bounding directory. The command can only copy files inside
+this directory. The default is `target`. This flag cannot be specified for
```

Comment:
When you say "default is `target`", does it refers to the value of the target flag or the word `target`? (It reads as the word `target`, in which case we shall also list all the other enum values).

Reply:
Done. `target` should refer to the argument value. I changed `target` to `{target}`.